src/v3/src/mocks/response/idp/idx/identify/error-invalid-username.json

```diff
@@ -125,7 +125,7 @@
       },
       "type": "password",
       "key": "okta_password",
-      "id": "aut2h3ffszv3me6O31d7",
+      "id": "aut1egri5t7iomJ3l1d7",
```

Comment:
Had to update this id to match the original response to pass flow scenarios (IDs do not change between flows).